operator/cmd/operator/server.go

```diff
@@ -117,7 +133,10 @@ func run() {
 		log.Warn("Leader election namespace not set. Leader election is disabled. NOT APPROPRIATE FOR PRODUCTION USE!")
 	}
 
+	// TODO renewDeadline cannot be greater than leaseDuration
 	renewDeadline := getRenewDeadline()
+	leaseDuration := getLeaseDuration()
```

Comment:
Does it make sense to make it relative to renewDeadline? like half or 2x or something?

Reply:
Open to suggestions.  Currently the dependent go package causes the operator to fail to start which is no good.  
1. No `LEASE_DURATION` configuration option exposed and lease duration is just always set 2x larger than renew deadline
2. Expose `LEASE_DURATION` configuration option but log a `WARN` when renew deadline is larger than lease duration and instead revert `RENEW_DEADLINE` back to default 10s value
3. something else

Reply:
From what I understand the ratio of LEASE_DURATION to RENEW_DEADLINE depends on how much you want to tolerate clock skew rate https://pkg.go.dev/k8s.io/client-go/tools/leaderelection

Having Lease Duration default to a set ratio (especially 2x) leverages the upstream knowledge of how much skew rate the operator needs to tolerate.

Having Lease Duration configurable relies on the user knowledge of how much skew rate the operator needs to tolerate.

I think either option 1 or 2 are solid picks. I lean towards option 1 since 2x tolerance is a solid margin already.

Reply:
defaults in the dependent go library are 10s and 15s, so I will go with option 1 and use a factor of 1.5x to remain what exists currently.  Fewer configurable environment variables sounds like the way to go